theme.json

```diff
@@ -57,38 +59,64 @@
 		"typography": {
 			"fontFamilies": [
 				{
-					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
-					"name": "System Font",
-					"slug": "system-font"
-				}
+					"fontFace": [
+						{
+							"fontDisplay": "block",
+							"fontFamily": "league_gothicregular",
+							"fontStretch": "normal",
+							"fontStyle": "normal",
+							"fontWeight": "200 800",
+							"src": [
+								"file:./assets/fonts/leaguegothic-regular-variablefont_wdth-webfont.woff2"
```

Comment:
I wonder if we should try to shorten the font name? Perhaps we could eliminate the type in `wdth` by just removing it.